apps/betterangels/package.json

```diff
@@ -79,7 +79,8 @@
     "expo-file-system": "*",
     "expo-sharing": "*",
     "@teovilla/react-native-web-maps": "*",
-    "react-native-paper": "*"
+    "react-native-paper": "*",
+    "@types/google.maps": "*"
```

Comment:
I think this is a dev dependency.

Reply:
So after i reinstall it as a devDep, `sync-deps` still puts it back under regular dependencies (in this file). 
Also, is there a process/command for uninstalling a package so it unsyncs it? Running sync-depds after removing it didn't do anything.

Reply:
let's take a look today after standup.  If it's automatically doing it then i guess we don't have a choice.

Reply:
done. i moved it manually